package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "typeorm",
   "version": "0.3.22",
-  "private": true,
```

Comment:
This was a safeguard to avoid accidentally publishing the wrong build. Could we undo this?

Reply:
It didn't work with it, I'll see if I can somehow bypass it